be/src/runtime/load_channel.cpp

```diff
@@ -85,12 +84,6 @@ LoadChannel::~LoadChannel() {
         rows_str << ", index id: " << entry.first << ", total_received_rows: " << entry.second.first
                  << ", num_rows_filtered: " << entry.second.second;
     }
-    if (_need_release_memtracker) {
```

Comment:
How count load channel memory now?

Reply:
This  PR does not affect how to count low channel memory, it only changed how to remove memtracker from workload group.